hibernate-rx-core/src/main/java/org/hibernate/rx/persister/impl/RxSingleTableEntityPersister.java

```diff
@@ -301,7 +298,7 @@ private void preInsertInMemoryValueGeneration(
 		return insertStage;
 	}
 
-	public CompletionStage<RowSet> insertRx(
+	public CompletionStage<?> insertRx(
```

Comment:
If this returns a `CS<Integer>` why use `?`?

Reply:
I'm still not sure it should return an integer or a specific class. I didn't want to commit to something at this stage. Mostly because it makes it easier to change thing at this stage.

Reply:
OK